api/pkg/apis/v1alpha1/providers/target/helm/helm.go

```diff
@@ -710,12 +745,25 @@ func configureUpgradeClient(componentProps *HelmChartProperty, deployment *model
 	upgradeClient.PostRenderer = postRenderer
 	// We can't add labels to the release in the current version of the helm client.
 	// This should added when we upgrade to helm ^3.13.1
-	return upgradeClient
+	return upgradeClient, nil
 }
 
-func configureUninstallClient(deployment *model.DeploymentSpec, config *action.Configuration) *action.Uninstall {
+func configureUninstallClient(componentProps *HelmChartProperty, deployment *model.DeploymentSpec, config *action.Configuration) (*action.Uninstall, error) {
 	uninstallClient := action.NewUninstall(config)
-	return uninstallClient
+	uninstallClient.Wait = componentProps.Wait
+	if componentProps.Timeout != "" {
+		duration, err := time.ParseDuration(componentProps.Timeout)
```

Comment:
Better to write a common function for parsing duration.

Reply:
OK， thanks jiawei